### PR TITLE
Update icepak.py

### PR DESCRIPTION
**Pull request by PipKat**

### CODE REVIEW

File: pyaedt/icepak.py [hunk 1]
```diff
@@ -1118,24 +1121,24 @@ def get_radiation_settings(self, radiation):
...
-        * Force Source Design Simulation, True or False
-        * Preserve Source Design Solution, True or False
-
+        """Retrieve a list of linked data.
+                
         Parameters
         ----------
         linkData : list
-            List containing the data in the link description.
+            List of the data to retrive for links. Options are:
```
Comment (PipKat) [inline pyaedt/icepak.py:1129]:
```suggestion
            List of the data to retrieve for links. Options are:
```

File: pyaedt/icepak.py [hunk 2]
```diff
@@ -1540,18 +1543,17 @@ def create_temp_point_monitor(self, point_name, point_coord = [0,0,0]):
 
     @aedt_exception_handler
     def delete_em_losses(self, bound_name):
-        """Delete EM losses boundaries.
+        """Delete EM losses bounday.
```
Comment (PipKat) [inline pyaedt/icepak.py:1546]:
```suggestion
        """Delete EM losses boundary.
```

File: pyaedt/icepak.py [hunk 3]
```diff
@@ -15,66 +15,65 @@
...
+    NG : bool, optional
+        Whether to run AEDT in the non-graphical mode. The default 
+        is``False``, which launches AEDT in the graphical mode.  
     AlwaysNew : bool, optional
         Whether to launch an instance of AEDT in a new thread, even if
         another instance of the ``specified_version`` is active on the
         machine.  The default is ``True``.
     release_on_exit : bool, optional
-        Whether to release AEDT on exit. The default is ``True``.
+        Whether to release  AEDT on exit.
```
Comment (akaszynski) [inline pyaedt/icepak.py:50]:
```suggestion
        Whether to release AEDT on exit. 
```

File: pyaedt/icepak.py [hunk 4]
```diff
@@ -603,47 +601,48 @@ def create_parametric_fin_heat_sink(self, hs_height=100, hs_width=100, hs_baseth
...
+            Pitch of the heat sink. The default is ``10``.
         thick : optional
-            The default value is ``10``.
+            Thickness of the heatsink. The default is ``10``.
         length: optional
-            The default value is ``40``.
+            The default is ``40``.
         height : optional
-            The default value is ``40``.
+            The default is ``40``.
```
Comment (akaszynski) [inline pyaedt/icepak.py:616]:
```suggestion
            Height of the heatsink.  The default is ``40``.
```

File: pyaedt/icepak.py [hunk 5]
```diff
@@ -603,47 +601,48 @@ def create_parametric_fin_heat_sink(self, hs_height=100, hs_width=100, hs_baseth
...
         draftangle : optional
-            The default value is ``0``.
+            The default is ``0``.
```
Comment (akaszynski) [inline pyaedt/icepak.py:618]:
```suggestion
            Draft angle.  The default is ``0``.
```

File: pyaedt/icepak.py [hunk 6]
```diff
@@ -603,47 +601,48 @@ def create_parametric_fin_heat_sink(self, hs_height=100, hs_width=100, hs_baseth
...
         patternangle : optional
-            The default value is ``10``.
+            The default is ``10``.
```
Comment (akaszynski) [inline pyaedt/icepak.py:620]:
```suggestion
            Pattern angle.  The default is ``10``.
```

File: pyaedt/icepak.py [hunk 7]
```diff
@@ -603,47 +601,48 @@ def create_parametric_fin_heat_sink(self, hs_height=100, hs_width=100, hs_baseth
...
         separation : optional
-            The default value is ``5``.
+            The default is ``5``.
         symmetric : bool, optional
-            The default value is ``True``.
+            The default is ``True``.
```
Comment (akaszynski) [inline pyaedt/icepak.py:624]:
```suggestion
            Whether the heatsink is symmetric.  The default is ``True``.
```

File: pyaedt/icepak.py [hunk 8]
```diff
@@ -603,47 +601,48 @@ def create_parametric_fin_heat_sink(self, hs_height=100, hs_width=100, hs_baseth
...
-            The default value is ``10``.
-        matname : optional
-            The default value is ``Al-Extruded``.
-        center : optional
-           The default value is ``[0,0,0]``.
+            The default is ``10``.
+        matname : str, optional
+            Name of the material. The default is ``Al-Extruded``.
+        center : list, optional
+           The default is ``[0,0,0]``.
```
Comment (akaszynski) [inline pyaedt/icepak.py:634]:
```suggestion
           Center of the heatsink.  The default is ``[0, 0, 0]``.
```

File: pyaedt/icepak.py [hunk 9]
```diff
@@ -1091,7 +1094,7 @@ def export_summary(self, output_dir=None, solution_name=None, type="Object", geo
 
     @aedt_exception_handler
     def get_radiation_settings(self, radiation):
-        """
+        """Retrieve radiation settings.
 
         Parameters
         ----------
```
Comment (akaszynski) [inline pyaedt/icepak.py:1100]:
```suggestion
        ----------
        radiation : str
            One of the following:
            
            * ``"Nothing"``
            * ``"Low"``
            * ``"High"``
            * ``"Both"``
```

File: pyaedt/icepak.py [hunk 10]
```diff
@@ -1091,10 +1094,17 @@ def export_summary(self, output_dir=None, solution_name=None, type="Object", geo
...
+        radiation : str
+            One of the following:
+            
+            * ``"Nothing"``
+            * ``"Low"``
+            * ``"High"``
+            * ``"Both"``
         radiation :
```
Comment (akaszynski) [inline pyaedt/icepak.py:1108]:
```suggestion
```

File: pyaedt/icepak.py [hunk 11]
```diff
@@ -1118,24 +1128,24 @@ def get_radiation_settings(self, radiation):
...
-
+        """Retrieve a list of linked data.
+                
         Parameters
         ----------
         linkData : list
-            List containing the data in the link description.
+            List of the data to retrieve for links. Options are:
+            
+            * Project name, if None => Use Active Project
```
Comment (akaszynski) [inline pyaedt/icepak.py:1137]:
```suggestion
            * Project name, if ``None`` use the active project.
```

File: pyaedt/icepak.py [hunk 12]
```diff
@@ -1118,24 +1128,24 @@ def get_radiation_settings(self, radiation):
...
+            * Design name
+            * HFSS solution name, such as ``"HFSS Setup 1 : Last Adaptive"``
+            * Force source design simulation, True or False
+            * Preserve source design solution, True or False
```
Comment (akaszynski) [inline pyaedt/icepak.py:1141]:
```suggestion
            * Force source design simulation, ``True`` or ``False``.
            * Preserve source design solution, ``True`` or ``False``.
```

File: pyaedt/icepak.py [hunk 13]
```diff
@@ -1464,7 +1475,7 @@ def create_assign_region2PCBComponent(self, scale_factor = 1.0, restore_padding_
 
         Returns
         -------
-
+
```
Comment (akaszynski) [inline pyaedt/icepak.py:1477]:
```suggestion
        tuple
            Tuple containing the ``(x, y, z)`` distances of the region.
```

Comment (akaszynski) [reply pyaedt/icepak.py:1477]: @PipKat, ``tuple`` type is a new one for you.  Lists are "mutable" meaning you can add things to them.  Tuples are not.  Returns from python are a tuple you return it with commas.  For example, here we have ``return dis_x, dis_y, dis_z``, which returns a tuple with three floats.

File: pyaedt/icepak.py [hunk 14]
```diff
@@ -1504,20 +1514,19 @@ def create_assign_region2PCBComponent(self, scale_factor = 1.0, restore_padding_
...
-        """Create a temperature monitor for simulation.
+        """Create a temperature monitor for the simulation.
 
         Parameters
         ----------
         point_name : str
             Name of the temperature monitor.
         point_coord : list, optional
-            Coordinate of the point monitor. The default is ``[0,0,0]``.
+            Point coordinates for the temperature monitor. The default is ``[0,0,0]``.
```
Comment (akaszynski) [inline pyaedt/icepak.py:1524]:
```suggestion
            Point coordinates for the temperature monitor. The default is ``[0, 0, 0]``.
```
Remember to separate items in a list by adding a single space after each comma.

File: pyaedt/icepak.py [hunk 15]
```diff
@@ -1540,18 +1549,17 @@ def create_temp_point_monitor(self, point_name, point_coord = [0,0,0]):
 
     @aedt_exception_handler
     def delete_em_losses(self, bound_name):
-        """Delete EM losses boundaries.
+        """Delete EM losses boundary.
 
         Parameters
         ----------
         bound_name : str
-            Name of the boundary to delete
```
Comment (akaszynski) [inline pyaedt/icepak.py:1548]: Personally, I think "to delete" is fine here, but that's totally up to you as both work.

**Review by akaszynski — APPROVED** [review]
Approved with suggestions.

Docs are looking way better.

**Pull request merged**